tutorials/2d/particle_systems_2d.rst

```diff
@@ -178,21 +178,21 @@ means particles are drawn according to their emission order (default).
 ParticlesMaterial settings
 --------------------------
 
-.. Commented out as not implemented in 3.x for now.
-..
-   Direction
-   ~~~~~~~~~
-..
-   This is the base angle at which particles emit. Default is ``0`` (down):
-..
-   .. image:: img/paranim1.gif
-..
-   Changing it will change the emissor direction, but gravity will still affect them:
-..
-   .. image:: img/paranim2.gif
-..
-   This parameter is useful because, by rotating the node, gravity will
-   also be rotated. Changing direction allows them to be separated.
+Direction
+~~~~~~~~~
+
+This is the base direction at which particles emit. The default is ``X:1``
+``Y:0`` ``Z:0`` which makes particles emit to the right. However with the
+default settings particles will go straight down.
+
+.. image:: img/direction1.png
+
+For this property to be noticeable you need an initial velocity (a setting
+we'll go over later) greater than 0. I've set the initial velocity to 40
+here, you'll notice that particles emit toward the right, then go down
+because of gravity.
```

Comment:
Say no to "I" in documentation :slightly_smiling_face: 

```suggestion
For this property to be noticeable, you need an *initial velocity* greater than 0.
Here, we set the initial velocity to 40. You'll notice that particles now emit
more visibly towards the right, then go downwards because of gravity.
```